### PR TITLE
[FSSDK-8919] chore: prepare for release 4.0.0-beta

### PR DESCRIPTION
**Pull request by jaeopt**
## Summary
Prepare for release 4.0.0-beta

## Issues
- [FSSDK-8919](https://jira.sso.episerver.net/browse/FSSDK-8919)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+
+With ODP accounts integrated into Optimizely projects, you can build audiences using segments pre-defined in ODP. The SDK will fetch the segments for given users and make decisions using the segments. For access to ODP audience targeting in your Feature Experimentation account, please contact your Customer Success Manager.
+
+This version includes the following changes:
+
+* New API added to `OptimizelyUserContext`:
+
+        - `fetchQualifiedSegments()`: this API will retrieve user segments from the ODP server. The fetched segments will be used for audience evaluation. The fetched data will be stored in the local cache to avoid repeated network delays.
+
+        - When an `OptimizelyUserContext` is created, the SDK will automatically send an identify request to the ODP server to facilitate observing user activities.
```
Comment (mikechu-optimizely) [inline CHANGELOG.md:25]: This block is coming up as a code block in the render
https://github.com/optimizely/android-sdk/blob/prepare-4.0.0-beta/CHANGELOG.md

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+
+* New APIs added to `OptimizelyClient`:
+
+        - `sendODPEvent()`: customers can build/send arbitrary ODP events that will bind user identifiers and data to user profiles in ODP.
+
+        - `createUserContext()` with anonymous user IDs: user-contexts can be created without a userId. The SDK will create and use a persistent `VUID` specific to a device when userId is not provided.
```
Comment (mikechu-optimizely) [inline CHANGELOG.md:31]: This is also rendering as a code block.  Is that desired?

**Review by mikechu-optimizely — CHANGES_REQUESTED** [review]
Small change

File: CHANGELOG.md (hunk 2 above)
Comment (jaeopt) [reply CHANGELOG.md]: Copy-paste changed the format unexpectedly :) Fixed.

**Review by mikechu-optimizely — APPROVED** [review]
LGreatTM!

**Pull request merged**